src/StructuredDiscussions/SDTopic.php

```diff
@@ -162,6 +162,10 @@ public function getTopicOwner(): Title {
 		return $this->associatedTitle;
 	}
 
```

Comment:
Please add a comment explaining this method, and how it differs from `getTopicOwner()`.